CMakeLists.txt

```diff
@@ -926,39 +860,29 @@ IF(ALBANY_MPI)
   MESSAGE("-- CTest parallel call: ${PARALLEL_CALL_TEXT}")
 ENDIF(ALBANY_MPI)
 
-#Add option to run any pretest executables in a specific environment (mpiexec.hydra, srun, or other)
-IF (DEFINED ALBANY_PRETEST_EXEC)
-  MESSAGE("-- ALBANY_PRETEST_EXEC is ${ALBANY_PRETEST_EXEC}")
-ELSE()
-  # Not set in configure - set it to be empty
-  SET(ALBANY_PRETEST_EXEC "" CACHE STRING "Command to use to execute pretests")
-ENDIF()
-
-#Add option to specify path to desired SEACAS utils
-IF (DEFINED ALBANY_SEACAS_PATH)
-  MESSAGE("-- ALBANY_SEACAS_PATH is ${ALBANY_SEACAS_PATH}")
-ENDIF()
-
-# Add Albany test -- in the "tests" directory
-IF (ALBANY_LIBRARIES_ONLY OR DISABLE_ALBANY_TESTS)
-  IF (ALBANY_LIBRARIES_ONLY) 
-    MESSAGE("\nALBANY_LIBRARIES_ONLY has been set: turning off all tests.")
-  ENDIF()
-  IF (DISABLE_ALBANY_TESTS) 
-    MESSAGE("\nDISABLE_ALBANY_TESTS has been set: turning off all tests.")
-  ENDIF()
-  IF (ALBANY_PYTHON)
-    SET(ALBANY_PYTHON_TESTS FALSE)
-  ENDIF()
-ELSE()
+#Add option to disable all tests 
+OPTION(DISABLE_ALBANY_TESTS "Flag to disable tests" OFF)
+IF (DISABLE_ALBANY_TESTS)
+  MESSAGE("-- DISABLE_ALBANY_TESTS is ON.  Tests will not be built.")
+elseif (ALBANY_LIBRARIES_ONLY)
```

Comment:
Are `DISABLE_ALBANY_TESTS` and `ALBANY_LIBRARY_ONLY` still used? The purpos of the latter was to avoid building the `Albany` and `AlbanyAnalysis` executables.

Reply:
Yeah, they seem very correlated... If you disable tests, why would you build the executable? I can remove one